### PR TITLE
ci: build elixir images for ce only

### PR DESCRIPTION
**Pull request by id**
Fixes <issue-number>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/` dir
- [ ] For EMQX 4.x: `appup` files updated (execute `scripts/update-appup.sh emqx`)
- [x] For internal contributor: there is a jira ticket to track this change, and another jira tickt to track doc updates (if any)
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

### CODE REVIEW

File: .github/actions/docker-meta/action.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    required: true
+    type: string
+  otp:
+    required: true
+    type: string
+  elixir:
+    required: false
+    type: string
+    default: ''
+  build_elixir:
```
Comment (thalesmg) [inline .github/actions/docker-meta/action.yaml:19]: Nit: this is apparently not used

File: .github/actions/docker-meta/action.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  using: composite
+  steps:
+    - name: prepare for docker/metadata-action
+      id: pre-meta
+      shell: bash
+      run: |
+        emqx_name=${{ inputs.profile }}
+        img_suffix=${{ inputs.arch }}
+        img_labels="org.opencontainers.image.otp.version=${{ inputs.otp }}"
+        if [ -n ${{ inputs.elixir }} ]; then
```
Comment (thalesmg) [inline .github/actions/docker-meta/action.yaml:56]: Maybe should be `-z`?  Or should use `[[ ... ]]`?

Comment (id) [reply .github/actions/docker-meta/action.yaml]: I want this condition to work when elixir version is not empty

Comment (thalesmg) [reply .github/actions/docker-meta/action.yaml]: I did some quick tests, it seems that either option is needed to have that behavior:

```bash
$ [ -n ${X} ] && echo yep || echo nope
yep
$ X=''
$ [ -n ${X} ] && echo yep || echo nope
yep
$ X='1.13'
$ [ -n ${X} ] && echo yep || echo nope
yep
$ [[ -n ${X} ]] && echo yep || echo nope
yep
$ X=''
$ [[ -n ${X} ]] && echo yep || echo nope
nope
$ [ -z ${X} ] && echo yep || echo nope
yep
$ X='1.13'
$ [ -z ${X} ] && echo yep || echo nope
nope
```

Comment (id) [reply .github/actions/docker-meta/action.yaml]: Variable must be in quotes (added in the current version)
```
bash-3.2$ X=''
bash-3.2$ [ -n ${X} ] && echo yep || echo nope
yep
bash-3.2$ X=''
bash-3.2$ [ -n "${X}" ] && echo yep || echo nope
nope
bash-3.2$ X='x'
bash-3.2$ [ -n "${X}" ] && echo yep || echo nope
yep
```

Comment (thalesmg) [reply .github/actions/docker-meta/action.yaml]: You're right, with quotes it works!  Ignore me.  :smiley_cat:

**Review by thalesmg — APPROVED** [review]

**Pull request merged**